sklearn/decomposition/_truncated_svd.py

```diff
@@ -6,6 +6,7 @@
 #         Michael Becker <mike@beckerfuffle.com>
 # License: 3-clause BSD.
 
+import numbers
```

Comment:
Would it be better to do `from numbers import Integral` here?

Reply:
I prefer using `from numbers import Integral`, but either way works for me.

Reply:
Yeah me too.  Changed.

Did I put the import in the right place?

Reply:
Yup looks good to me.